sql.txt

```diff
@@ -1,14 +1,40 @@
 (1) Using the Customer Table, select the CustomerName, ContactName, and Country.  Use ORDER BY to order by Country.  
 Use LIMIT and OFFSET to get entries 11 through 20. Paste your SQL statement below. 
+SELECT CustomerName, ContactName, Country 
+FROM Customers 
+ORDER BY Country 
+LIMIT 10 
+OFFSET 10;
 
 (2) Select all columns from the Customer table where the ContactName starts with A.  Paste your SQL statement below.
+SELECT * 
+FROM Customers
+WHERE ContactName LIKE 'A%';
 
 (3) Select all columns from the OrderDetails table where the ProductID is 51 and the quantity is greater than 10.
 Paste your SQL statement below.
 
+SELECT * 
+FROM OrderDetails 
+WHERE ProductID=51
+AND Quantity > 10;
+
 (4) Insert 3 rows into the Products table.  Note that you will have to specify a valid SupplierID and CategoryID, 
 corresponding to rows from the Supplier and Category tables.  Paste your three SQL statements below.
+INSERT INTO Products(ProductName, SupplierID, CategoryID, Unit, Price)
+    VALUES ('Product1', 1, 1, "500 ml", 10),
+           ('Product2', 2, 2, "5 kg pkg.", 10),
+           ('Product3', 3, 3, "12 boxes", 20);
 
 (5) Update the two top rows of the Products Table to increase the price by 1.50. (Get SQL to do the addition for you.)  Paste your SQL statement below.
 
+UPDATE Products
+SET Price=Price+1.50
+WHERE ProductID IN (
+    SELECT ProductID 
+    FROM Products
+    LIMIT 2);
+
```

Comment:
Way to use a subquery! If you're looking to shorten it, you could also consider:
```
UPDATE Products
SET Price = Price + 1.50
WHERE ProductID < 3;
```